plugins/jobs/src/js/JobsMenuClient.js

```diff
@@ -0,0 +1,28 @@
+import "rxjs/add/operator/combineLatest";
+import "rxjs/add/operator/map";
+
+import { makeExecutableSchema } from "graphql-tools/dist/index";
+import { runJob } from "#SRC/js/events/MetronomeClient";
+
+export const typeDefs = `
+  type Job {
+      jobId: String
+  }
+
+  type Mutation {
+    runJob(jobId: String!): Job!
```

Comment:
The return type is `JobRun` and not Job. For a job the id would be just id. As I am currently working on typing this stuff out and transforming it into something with the shape of a struct I think a renaming here would be suficcient. Otherwise we would do double work if you would adapt the schema here and did the data transformation

Reply:
I changed the type to JobRun and the ids to id.